src/pools/abstract-pool.ts

```diff
@@ -199,6 +198,20 @@ export abstract class AbstractPool<
     }
   }
 
+  /**
+   * Increase the number of tasks that the given workers has done.
+   *
+   * @param worker Workers whose tasks are increased.
+   */
+  protected decreaseWorkersTasks (worker: Worker): void {
```

Comment:
JSDoc and function name doesn't match each other? Increase or Decrease?

Reply:
Decrease thank you to catch